ChatSX/components/InputBox/index.tsx

```diff
@@ -224,67 +212,76 @@ const InputBox = ({
     });
   };
 
+  // this method sets image and video to upload on FileUpload screen via redux.
+  const handleImageAndVideoUpload = async (selectedImages: Asset[]) => {
+    if (!!selectedImages) {
+      if (isUploadScreen === false) {
+        await handleVideoThumbnail(selectedImages);
+        dispatch({
+          type: SELECTED_FILE_TO_VIEW,
+          body: {image: selectedImages[0]},
+        });
+        dispatch({
+          type: STATUS_BAR_STYLE,
+          body: {color: STYLES.$STATUS_BAR_STYLE['light-content']},
+        });
+      } else if (isUploadScreen === true) {
+        // selected files will be saved in redux inside get video function
+        const res = await getVideoThumbnail({
+          selectedImages,
+          selectedFilesToUpload,
+          selectedFilesToUploadThumbnails,
+          initial: false,
+        });
+        dispatch({
+          type: SELECTED_FILES_TO_UPLOAD_THUMBNAILS,
+          body: {
+            images: res?.selectedFilesToUploadThumbnails,
+          },
+        });
+        dispatch({
+          type: SELECTED_FILES_TO_UPLOAD,
+          body: {
+            images: selectedImages[0],
+          },
+        });
+      }
```

Comment:
`else` case is missing? we have mentioned `if` and `else-if`, `else` is missing?

Reply:
There is no need for `else` case, no need to handle `undefined` case as `isUploadScreen` is `boolean` either it can be `false` or `true` which has been handled